src/GlobalScreen/Scope/MetaBar/Factory/TopLegacyItem.php

```diff
@@ -65,11 +66,12 @@ public function hasSymbol() : bool
     /**
      * @inheritDoc
      */
-    public function withTitle(string $title) : hasTitle
+    public function withTitle(string $title, ?string $title_language = null) : hasTitle
```

Comment:
Some think here... I currently think of an Interface which defines those Methods at once. A trait implementeting them would reduce the duplicate code. I do not think all `hasTitle` items has the title_language?